.travis.yml

```diff
@@ -1,12 +1,24 @@
+---
 language: ruby
 sudo: false
 rvm:
   - 1.9.3
-  - 2.0.0
-  - 2.1.8
-  - 2.2.4
-  - 2.3.0
-  - rbx-19mode
-  - jruby-19mode
+  - 2.0
```

Comment:
ruby 2.2 and earlier is no longer officially supported, so no need to keep them in the test matrix